examples/rhdh-cr-with-app-configs.yaml

```diff
@@ -19,9 +19,14 @@ spec:
       mountPath: /tmp/my-extra-files
       configMaps:
         - name: "my-backstage-extra-files-cm1"
+        # RHIDP-2246: mounting secret or configmap with a '.' in the name
```

Comment:
Do we really need to maintain the code for this case in, e2e, integration and unit tests?
Can not it be simply covered with unit tests only instead to simplify maintainability?

Reply:
As discussed, I think it makes sense to keep at least the unit and integration tests. I've removed the changes to this example.
[`aedd2d1` (#363)](https://github.com/janus-idp/operator/pull/363/commits/aedd2d14ad22a901899e9d18c40af252ce890445)